src/libmongoc/src/mongoc/mongoc-openssl.c

```diff
@@ -817,7 +818,8 @@ _mongoc_ocsp_tlsext_status (SSL *ssl, mongoc_openssl_ocsp_opt_t *opts)
    }
 
    /* Get the stapled OCSP response returned by the server */
-   len = SSL_get_tlsext_status_ocsp_resp (ssl, &resp_data);
+   len = SSL_get_tlsext_status_ocsp_resp (ssl, &mutable_resp_data);
```

Comment:
Note for future readers: Many SSL_ APIs macro-expand to an `SSL_ctrl` invocation that accepts a bare non-`const` opaque `void*`. This may or may not generate a warning since `p const* *` is a non-`const` pointer-to `const p`.